### ISSUE TITLE
使用adb访问sdcard/Android/data

### ISSUE THREAD

**Issue by for-the-zero** [zhanghai/MaterialFiles#1132]
不一定要root，可以用adb

建议通过shizuku启用

![Screenshot_20240214_103014](https://github.com/zhanghai/MaterialFiles/assets/101497510/b5d4a085-c7d1-4bd2-9605-16841095f6de)

**Comment by Freed-Wu** [comment]
Yes, https://shizuku.rikka.app/ allow app to get (adb) shell permission. For some paths (QQ, wechat storage path), it is enough.

**Comment by zhanghai** [comment]
Feel free to create a DocumentsProvider app using Shizuku, and then you can use Material Files to browse that provider.
